### DISCUSSION TITLE
Many problems with new version

### DISCUSSION THREAD

**Discussion by ravananga** [aclap-dev/video-downloadhelper#167] [Help and Q&A]
### Your question

- If a download was canceled due to an error, only the (incomplete) file can be played in the menu but not downloaded again (menu item "Download" removed in new version).
- Downloads cannot be canceled.
- Even if "Questions" is set, the download helper does not ask which application should be used.
- How can I downgrade?
- I do not have a credit card to support Downloadhelper.

Best regards

### Addon & CoApp version and details

Version 8.2.0.23 (stable)
Begleitanwendung gefunden: VdhCoApp 2.0.17

**Comment by paulrouget** [comment]
Right, some of these issues have been fixed in version 8.2.1.1 which should autoupdate soon!

**Comment by ravananga** [comment]
Thanks!
